DOCS/history-OSI-SPDX.md

```diff
@@ -0,0 +1,87 @@
+This is probably because these licenses were originally on the SPDX License List as a stand alone licenses. However, when SPDX 2.0 came out in May 2015, exceptions were moved to their own part of the SPDX License List to be used with the `WITH` operator to allow more extensible and varied license expressions, see https://spdx.org/licenses/eCos-2.0.html and now see https://spdx.org/licenses/eCos-exception-2.0.html
+and https://spdx.org/licenses/wxWindows.html and now see https://spdx.org/licenses/WxWindows-exception-3.1.html
+
+Note that OSI has the GPL-2.0-or-later license notice on its page for the eCos exception, which would use the SPDX license expression: `GPL-2.0-or-later WITH eCos-exception-2.0`. Did OSI _only_ approve this exception for GPL-2.0-or-later? 
+
+Suggestion: Update the pages to use the current SPDX identifier for the exception (or the full SPDX license expression). Clarify what versions of GPL it is approved for.
+
+## Licenses that allow for variations by way of a different license notice
+These are licenses that were approved by the OSI in full. These licenses allow legally substantive variations based on use of a different license notice. This variataion warranted distinct SPDX identifiers to indicate whether or not the allowed variation is being triggered. All of these variations include a Note in the SPDX License List entry explaining this. 
+
+* `CAL-1.0-Combined-Work-Exception`
+* `MPL-2.0-no-copyleft-exception`
+* `OFL-1.1-no-RFN`
+* `OFL-1.1-RFN`
+* GNU license family, see https://github.com/OpenSourceOrg/licenses/issues/87
+
+Suggestion: OSI could add the variant SPDX ids to the license pages.
+
+## Not in SPDX list, but in OSI's list
+
+* [Jabber Open Source License](https://opensource.org/licenses/jabberpl) - This is a case where the text on the OSI site is not the same as the archived text at http://archive.jabber.org/core/JOSL.pdf SPDX would consider these two texts as different licenses, and was trying to clarify with the OSI which text was submitted and approved by OSI. Martin Michlmayr and Jilayne discussed this back in 2013 (https://lists.spdx.org/g/Spdx-legal/topic/22080295#475), but never reached a resolution. Jilayne emailed the OSI Board about this issue in July 2019. 
+
+Suggestion: SPDX could recognize both variants on the SPDX License List and note the one that is specifically OSI-approved. This would require clarification from OSI as to which one was approved and make any updates to its site accordingly. If both variants were OSI-approve, then a new entry would be warranted on the OSI site. 
+
+* [CVW: MITRE Collaborative Virtual Workspace License](https://opensource.org/license/cvw) - Needs further research as to when it was OSI approved and how it didn't get on SPDX License List as well.
+
+## Other issues
+* [The W3C® SOFTWARE NOTICE AND LICENSE](https://opensource.org/license/w3c) - uses SPDX id, `W3C-20150513` but licnese text does not seem to match??
```

Comment:
"licnese" => "license"

Also, is there a further action intended to be taken here, or is this just recording it as an open-ended question?

Note that #2165 appears to be asking about this. Would it be more appropriate to track in that issue thread?

Reply:
agreed, I've removed this